app/helpers/friendships_helper.rb

```diff
@@ -0,0 +1,2 @@
+module FriendshipsHelper
```

Comment:
**Kindly work on these changes:**
- Provide a Link to Heroku in README file.
- Should have merged a PR from the development branch to the main/master branch with all changes made by them in previous milestones.
- Provide a Link to the deployed app is working.